flink-runtime/src/test/java/org/apache/flink/runtime/io/network/netty/NettyPartitionRequestClientTest.java

```diff
@@ -142,6 +147,45 @@ public void testDoublePartitionRequest() throws Exception {
 		}
 	}
 
+	@Test
+	public void testResumeConsumption() throws Exception {
+		final CreditBasedPartitionRequestClientHandler handler = new CreditBasedPartitionRequestClientHandler();
+		final EmbeddedChannel channel = new EmbeddedChannel(handler);
+		final PartitionRequestClient client = createPartitionRequestClient(channel, handler);
+
+		final NetworkBufferPool networkBufferPool = new NetworkBufferPool(10, 32, 2);
+		final SingleInputGate inputGate = createSingleInputGate(1);
+		final RemoteInputChannel inputChannel = createRemoteInputChannel(inputGate, client, networkBufferPool);
+
+		try {
+			client.resumeConsumption(inputChannel);
```

Comment:
is it possible to call `inputChannel#resumeConsumption` instead of `client.resumeConsumption`, then we can further verify the interaction between input channel and client.